.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,20 @@
+name: golangci-lint
```

Comment:
Would it be better to split the CI workflow into a separate commit on top of fixing the lint errors ?

Reply:
IMHO it would be great to at least split the cosmetic/noop changes (whitespace, semi-colon) from the functional ones.
Using a `WriteDirectory` loop in `startup` would also be nice to have.

Regardless of the above, as a whole I'm really glad to see this PR. Thank you

Reply:
None of this is done yet;  I created the PR to test whether the workflow worked as expected ; switched to draft status to make that more clear